tests/Benchmarks/Utf8String/CodePointEnumeration.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Code;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using Utf8StringRealType = System.Text.Utf8.Utf8String;
+
+namespace Benchmarks.Utf8String
+{
+    public partial class Utf8String
```

Comment:
Rename this class to CodePointEnumeration and then we can remove `Utf8StringRealType`